.eslintignore

```diff
@@ -1,5 +1,6 @@
 /fixtures/
 node_modules/
+pnpm-lock.yaml
```

Comment:
This is needed so that `pnpm format` doesn't format the lock file. We ran into an issue in the Remix repo where CI got stuck in a loop deduping and formatting `pnpm-lock.yaml` since pnpm and Prettier didn't agree, so this ensures we don't hit that problem again.